### PR TITLE
Fixing the issue #2791 (Export backups to a sub-folder)

### PR DESCRIPTION
**Pull request by frankabjr**
This pr puts the plain text backup under the folder Signal/Backup/. On the import side, verify if old backup file exist (to oldest clients). Also, I've changed the EncryptedBackupExporter, to point to the new backup path. 

I didn't have tested the EncryptedBackupExporter because this is not working.

### CODE REVIEW

File: src/org/thoughtcrime/securesms/database/EncryptedBackupExporter.java [hunk 1]
```diff
@@ -26,15 +26,23 @@
...
 
-public class EncryptedBackupExporter {
+public class EncryptedBackupExporter extends AbstractBackup {
+
+  public static void setup() {
+    File backupDirectory = getDirectoryPath();
+    if (!backupDirectory.exists()) {
+      backupDirectory.mkdirs();
+    }
+  }
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/database/EncryptedBackupExporter.java]: classes should not have public methods called setup() that take no arguments and return no value.  that's what a constructor is for, but in this case it seems like all you need to do is call mkdirs() when you are returning the value

Comment (frankabjr) [inline src/org/thoughtcrime/securesms/database/EncryptedBackupExporter.java]: I got it. Fixed in the commit 10314d0.

**Comment by moxie0** [comment]
two space indents

**Comment by frankabjr** [comment]
Ok. I'll close this pr and create a new with all changes in only one commit.

**Pull request closed without merging**